dll/win32/shell32/wine/shellpath.c

```diff
@@ -103,6 +103,13 @@ DoGetProductType(PNT_PRODUCT_TYPE ProductType)
     return TRUE;
 }
 
+BOOL APIENTRY IsRemovableDrive(DWORD iDrive)
+{
+    WCHAR szRoot[] = L"C:\\";
+    szRoot[0] = (WCHAR)(L'A' + iDrive);
```

Comment:
No range check on `iDrive` ?

Reply:
Windows doesn't check the value. Should we add an assertion for it?

Reply:
Added.